### ISSUE TITLE
Feature request: Add shadow behind clock

### ISSUE THREAD

**Issue by tasmo** [MM2-0/Kvaesitso#1125]
When I set the theme colors to be taken from the wallpaper sometimes the clock is hard to read. If the wallpaper is mainly dark but the area behind the clock can be as light is the clock itself.

An option to add (user defined) shadow would be great

**Comment by NicCrimson** [comment]
Yes, this would be great. It blends in with certain wallapers to much.
